src/collections/_documentation/workflow/alerts-notifications/alerts.md

```diff
@@ -0,0 +1,187 @@
+    - interval: one minute, one hour, one day, one week, or 30 days
+- Sentry detects a new event
+- Sentry detects a new event with tags matching {key} {match} {value}
+    - key: any tag [LINK: tags]
+    - match: equals, does not equal, starts with, ends with, contains, does not contain, is set, or is not set
+    - value: any key’s value
+- Sentry detects a new event with {attribute} {match} {value}
+    - attribute: `message`, `platform`, `environment`, `type`, `exception.type`, `exception.value`, `user.id`, `user.email`, `user.username`, `user.ip_address`, `http.method`, `http.url`, `stacktrace.code`, `stacktrace.module`, or `stacktrace.filename`
+    - match: equals, does not equal, starts with, ends with, contains, does not contain, is set, or is not set
+    - value: any attribute’s value
+- Sentry detects a new event with level {match} {level}
+    - match: equal to, less than or equal to, greater than or equal to
+    - level: fatal, error, warning, info, debug, or sample
+
+### **Actions**
+
+The following actions are available:
+
+- Send a notification to all legacy integrations
+    - Legacy integrations [LINK: legacy integrations], also known as Plugins, are configured per project. Legacy integrations currently include email.
+- Send a notification to a single legacy integration (service)
+    - "service" is the same as legacy integration
+- Send a notification to a global integration, which includes
+    - [PagerDuty](https://docs.sentry.io/workflow/integrations/global-integrations/#pagerduty)
+    - [Slack](https://docs.sentry.io/workflow/integrations/global-integrations/#slack)
+    - Custom integrations built using the [Integration Platform](https://docs.sentry.io/workflow/integrations/integration-platform/)
+
+![DON%20T%20add%20content%20to%20this%20Draft%20Alerts%20Notificatio/Screenshot_2020-02-25_15.28.01.png](DON%20T%20add%20content%20to%20this%20Draft%20Alerts%20Notificatio/Screenshot_2020-02-25_15.28.01.png)
+
+- Legacy Integrations (via “Service” as any individual integration) that can send Notifications:
+
+    [ ALERT BOX: **Community Integrations** 
+
+    These integrations are [maintained and supported](https://forum.sentry.io/) by the Sentry community. ]
+
+    - Campfire*
+    - IRC
+    - OpsGenie
+    - Pushover
+    - Twilio
+    - VictorOps
+    - [Webhooks](https://docs.sentry.io/webhook-plugin/)
+
+By default, [LINK: issue owners] receive alert emails. If an issue owner is not configured or found, the email will either not send or send to all project members as defined in **Project Settings > Issue Owners**.
+
+![DON%20T%20add%20content%20to%20this%20Draft%20Alerts%20Notificatio/Screenshot_2020-02-26_22.03.07.png](DON%20T%20add%20content%20to%20this%20Draft%20Alerts%20Notificatio/Screenshot_2020-02-26_22.03.07.png)
+
+**Rate Limit**
+
+Rate Limit control allows you to limit the number of times actions execute for each issue. The limit is set to perform the action according to one of these intervals:
```

Comment:
So this is a bit more than just frequency of actions.

"The rate limit determines how frequently an issue alerts. After an issue fires an alert, conditions will not be checked and actions will not be executed for that issue until the rate limit period passes."